mne/io/tests/test_raw.py

```diff
@@ -88,6 +89,14 @@ def _test_raw_reader(reader, test_preloading=True, **kwargs):
     idx = np.where(concat_raw.annotations.description == 'BAD boundary')[0]
     assert_array_almost_equal([(last_samp - first_samp) / raw.info['sfreq']],
                               concat_raw.annotations.onset[idx], decimal=2)
+
+    if raw.info['meas_id'] is not None:
+        for key in ['secs', 'usecs', 'version']:
+            assert_equal(raw.info['meas_id'][key], raw3.info['meas_id'][key])
+        assert_array_equal(raw.info['meas_id']['machid'],
+                           raw3.info['meas_id']['machid'])
+    else:
+        logger.warn('Reader does not set meas_id')
```

Comment:
Do we want this to raise an error? it would catch IO readers that don't set meas_id.

Currently it is only raising errors when the reader doesn't set meas_id and when the tests that expect the warning messages  to be a specific thing.

Reply:
We probably don't need to warn. I don't think we are guaranteed that all formats will have it.

Reply:
Ok I can pull it. What is info['meas_id'] used for... anything?